ansible/roles/products/common/docker-container/tasks/create-container/no-setup-with-terminator.yml

```diff
@@ -26,5 +26,5 @@
     - /tmp/.X11-unix:/tmp/.X11-unix[rw]
     - /dev/input:/dev/input[rw]
     - /run/udev/data:/run/udev/data
-    command: terminator -x bash
+    command: terminator --title='SERVER CONTAINER' -x bash
```

Comment:
Inform @mikramarc as he is doing something similar with naming windows

Reply:
I have informed him